apps/myaccount/src/components/profile/profile.tsx

```diff
@@ -293,6 +307,43 @@ export const Profile: FunctionComponent<ProfileProps> = (props: ProfileProps): R
         setCountryList(ReusableCommonUtils.getCountryList());
     }, []);
 
+    /**
+     * API call to get validation configurations.
+     */
+    const getConfigurations = (): void => {
+
+        fetchPasswordValidationConfig()
+            .then((response: ValidationFormInterface[]) => {
+                setUsernameConfig(getUsernameConfiguration(response));
+            })
+            .catch((error: any) => {
+                if (error.response && error.response.data && error.response.detail) {
```

Comment:
Shouldn't this be `error.response.data.detail` rather `error.response.detail`?

In that case, minify this as follows with null check.
```suggestion
                if (error?.response?.data?.detail) {
```

Reply:
Addressed with 2255de611fd68c6c9f2029e0bfd1dbab1c90e8c3.